### PR TITLE
Creating project-level bib file and use that bib file if possible

### LINKED ISSUE #20: Can we use master files (i.e. bib, tex, logos) across all templates
We do not want to have the same file multiple places in the package if we can avoid it

This could also apply to methods or other writing sections that would be consistent and not project specific. Protocol colors and group names is a different issue and those files live in the protocol folder, where these files live in the package (at least initially).

### PR DESCRIPTION
**Pull request by wfulp**
1. A single bib file is saved in `VISCtemplates ` package **_inst/templates_** folder, and gets saved into the users _**docs**_ folder when running `create_visc_project()`. The bib file is not saved with each report anymore.
2. When a user knits a report with either `visc_pdf_document` or `visc_word_document` output it checks for a project-level bib file (docs/bibliography.bib), and only creates a report-level bib file if a project-level bib does not exist. (Report-level Readme file also gits created)
3. The skeleton prioritizes a report-level bib file if both project and report-level bib files exist (but this would only happen if the user manually adds a report-level bib file).

I also paired down the the bib file dramatically because the large file was not properly getting saved in the new project through `create_visc_project()` (`usethis::use_template` was failing on the old bib file).

Resolves #20 (other parts of the issue moved over in PR #22)

### CODE REVIEW

**Comment by mayerbry** [comment]
Very nice! I didn't do a thorough code review, but I skimmed through and nothing stuck out to me.

File: R/template.R [hunk 1]
```diff
@@ -110,6 +123,17 @@ visc_pdf_document <- function(latex_engine = "pdflatex",
...
+  # If no project-level bib file creating report specific bib
+  if (!file.exists(here::here('docs', 'bibliography.bib')))
+    file.copy(from = system.file("templates", 'bibliography.bib',
+                                 package = "VISCtemplates"),
+            to = "bibliography.bib",
+            overwrite = FALSE)
+
+  file.copy(from = find_resource("visc_report", "README_PT_Report.md"),
+            to = "README.md",
+            overwrite = FALSE)
```
[the remark this answers is not in the document]
Comment (wfulp) [reply R/template.R:135]: It was, but the file was saved in the skeleton folder for each report. This way the readme lives where the logos lives, and only needs to be in one place, even if we have multiple skeletons using `visc_pdf_document` output.

File: inst/rmarkdown/templates/visc_report/template.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
-name: VISC Report (PDF & Word Output)
+name: VISC PT Report (Generic)
```
[the remark this answers is not in the document]
Comment (wfulp) [reply inst/rmarkdown/templates/visc_report/template.yaml:1]: Yeah I changed this for a better naming convention for the upcoming barebones (Empty) and assay specific templates. Should I move this to the barebones PR?

[the remark this answers is not in the document]
Comment (monicagerber) [reply inst/rmarkdown/templates/visc_report/template.yaml:1]: Ah, got it. Yes, I think it makes sense to move.

File: R/template.R (hunk 1 above)
[the remark this answers is not in the document]
Comment (monicagerber) [reply R/template.R:136]: I think I'm following, but not 100% sure what you mean.
"For each report" --> for both the PT report template and "bare bones" template?

[the remark this answers is not in the document]
Comment (wfulp) [reply R/template.R:136]: yes, as well as the Assay specific templates (unless we decide to write assay specific READMEs)

[the remark this answers is not in the document]
Comment (monicagerber) [reply R/template.R:136]: Ok, makes sense.

**Comment by wfulp** [comment]
The `utils` error is addressed in #48, so once that's merged into devel we should be good.

**Pull request merged**